charts/civil-sdt/values.preview.template.yaml

```diff
@@ -15,7 +15,14 @@ java:
     civil:
       resourceGroup: civil
       secrets:
-        - name: AppInsightsInstrumentationKey
+        - name: civil-sdt-service-s2s-secret
+          alias: CIVIL_SDT_S2S_KEY
+        - name: civil-sdt-POSTGRES-USER
+          alias: SDT_DB_USERNAME
+        - name: civil-sdt-POSTGRES-PASS
+          alias: SDT_DB_PASSWORD
+        - name: civil-sdt-servicebus-connection-string
+          alias: CIVIL_SDT_SERVICEBUS_CONNECTION_STRING
   secrets:
     CIVIL_SDT_SERVICEBUS_CONNECTION_STRING:
       secretRef: servicebus-secret-queue-{{ .Release.Name }}-servicebus-civil-sdt-in-out
```

Comment:
Servicebus values have been changed in main branch (for servicebus v1.0.4).  Another indication that this branch needs to be updated with changes from main branch!